srcpkgs/paleta/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'paleta'
+pkgname=paleta
+version=1.0.2
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkg-config tar"
+makedepends="cmake"
+short_desc="Change terminal colors on-the-fly independent of terminal emulator"
+maintainer="Brihadeesh <brihadeesh@protonmail.com>"
+license="MIT"
+homepage="https://github.com/dylanaraps/paleta"
+distfiles="https://github.com/dylanaraps/${pkgname}/archive/${version}.tar.gz"
```

Comment:
If you're going to expand in distfiles, it would be much better if you did: `distfiles="${homepage}/archive/${version}.tar.gz"`.

Reply:
Will incorporate this. Makes sense